docs/reference/remap.cue

```diff
@@ -169,19 +169,15 @@ remap: {
 			href: "functions"
 
 			description: """
-				In TRL, functions act just like they do in standard programming languages. They can
-				take inputs (or no input) and either return values or make an
-				[assertion](#assertion) using the [`assert`](#assert) function.
-
-				TRL types that can serve as inputs to functions are listed in [Parameter
-				types](#parameter-types); types that functions can output are listed in [Return
-				types](#return-types).
+				In TRL, functions can take inputs (or no input) and return a value, `null`, or an
+				error.
```

Comment:
Technically `null` _is_ a value, so not sure if there's a need to mention it separately here.